static_src/components/form.jsx

```diff
@@ -0,0 +1,257 @@
+
```

Comment:
This is great. I only have one worry, is there an existing react library that creates these kind of form elements already? Just trying to think long term since we would have to possibly add additional `FormElements` in the future